webpack.config.js

```diff
@@ -1,21 +1,35 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const path = require('path');
+
 module.exports = {
   entry: './src/index.tsx',
   output: {
     filename: 'DecreaseVolumeOnSpeak.js',
     library: 'DecreaseVolumeOnSpeak',
     libraryTarget: 'umd',
-    publicPath: '/static/',
+    publicPath: '/',
     globalObject: 'this',
   },
   devServer: {
     allowedHosts: 'all',
     port: 4701,
-    host: 'localhost',
+    host: '0.0.0.0',
     hot: false,
     liveReload: false,
     client: {
       overlay: false,
     },
+    onBeforeSetupMiddleware: (devServer) => {
```

Comment:
Why not https://webpack.js.org/configuration/dev-server/#devserversetupmiddlewares? Seems like `onBeforeSetupMiddleware` is being deprecated.